homeassistant/components/tuya/alarm_control_panel.py

```diff
@@ -45,6 +45,14 @@ class Mode(StrEnum):
 # All descriptions can be found here:
 # https://developer.tuya.com/en/docs/iot/standarddescription?id=K9i5ql6waswzq
 ALARM: dict[str, tuple[AlarmControlPanelEntityDescription, ...]] = {
+    # Multi-functional Sensor
+    # https://developer.tuya.com/en/docs/iot/categorydgnbj?id=Kaiuz3yorvzg3
+    "dgnbj": (
+        AlarmControlPanelEntityDescription(
+            key=DPCode.MASTER_MODE,
+            name="Alarm",
+        ),
+    ),
```

Comment:
A lot of sensors support "alarm" like functionalities in Tuya, but are they standalone or part of the generic alarm function of Tuya? As in, is this a control panel? Can you actively walk up to the device and arm it?

Reply:
according to the documentation within the category of Tuya Developer portal, yes...
In the specific device I have, no

Reply:
Ok, lets remove this part until we can test/confirm that.

Reply:
on it

Reply:
done